tests/language/expressions/bigint/bigint-comparison.js

```
@@ -0,0 +1,35 @@
+/*---
+description: BigInt comparison operations
+features: [bigint]
+---*/
+
+test("strict equality between BigInts", () => {
+  expect(1n === 1n).toBe(true);
```

Biome (2.4.11)

[error] Comparing to itself is potentially pointless.

(lint/suspicious/noSelfCompare)